### PR TITLE
refactor: rename erc20-relay to ethereum-event-client

### LINKED ISSUE #232: Rename `erc20-relay` to `ethereum-event-client`
### File Location(s)

_No response_

### Proposal

-

### PR DESCRIPTION
**Pull request by playX18**
Resolves #232

### CODE REVIEW

**Review by gshep — APPROVED** [review]

File: indexer/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM node:20-alpine AS node
 FROM node AS api
 WORKDIR /assets
 COPY api/gear/vft_manager.idl .
-COPY api/gear/erc20_relay.idl .
+COPY api/gear/ethereum_event_client.idl .
```
Comment (mertwole) [inline indexer/Dockerfile:6]: I suppose it isn't required in indexer anymore? `Relayed` event from `historical-proxy` should be enough

File: gear-programs/historical-proxy/app/src/service.rs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 // Incorporate code generated based on the IDL file
 
 #[allow(dead_code)]
-pub(crate) mod erc20_relay {
-    include!(concat!(env!("OUT_DIR"), "/erc20_relay.rs"));
+pub(crate) mod ethereum_event_client_ {
```
Comment (mertwole) [inline gear-programs/historical-proxy/app/src/service.rs:4]:
```suggestion
pub(crate) mod ethereum_event_client {
```

File: indexer/Dockerfile (hunk 1 above)
Comment (mertwole) [reply indexer/Dockerfile:6]: Ah, I see it was changed in #237, so I think you can revert changes to indexer in this PR and wait for #237 to be merged

**Review by mertwole — COMMENTED** [review]
Please don't merge it yet

**Review by mertwole — APPROVED** [review]

**Pull request merged**